### ISSUE TITLE
smooth

### ISSUE THREAD

**Issue by daasiankid** [w3x731/survivIoAim#31]
I feel like the movements of the bot are very jerky, is it possible to make it as smooth as butter? so it looks like a player playing not a bot?

**Comment by Kalaborative** [comment]
I agree, the [surviv god mode](https://chrome.google.com/webstore/detail/surviv-god-mode/gbefklfanaaomnllnheeihddnhopgdce) hack played much smoother. Whenever I activate it, gameplay in general just feels a lot slower and movement is quite jerky. Is there a way you could sort of integrate that?
